.gitlab-ci.yml

```
@@ -33,11 +36,23 @@ build:dist:
     paths:
       - dist.tar.gz
 
+test:static:
```

```suggestion
test:backend:static:
```
or maybe `test:golang:static` as we likely still have other static checks coming in

---

Actually, independent of this comment I was actually going to align this with #33. Thanks for the reminder :)